necroHeal.lic

```diff
@@ -0,0 +1,57 @@
+=begin
+
+  Documentation: https://elanthipedia.play.net/Lich_script_development#necroheal
+  ## ** Placeming comments here until finalized and added to pedia.
+
+  Necromancer stand-alone healing script
+  
+  Uses all settings defined in HEAL YAML FILE (default)
+  Consume logic via combat trainer
+  
+  TO DO:
+  - See pull request #1462 for harvested materials.
+  - See if there is a background way to use common-healing to determine wound status
+  - Add YAML variable to swap YAML file name
+  
+=end
+
+custom_require.call(%w(common common-healing events))
+
+
+  if DRCH.check_health.empty?
+    echo 'Not wounded, exiting heal script'
+    exit
+  end
+
+
+  ## Sets flag to check for game line to determine when no wounds exist
+  ## Flag uses consume logic from combat-training. If no wounds are left
+  ## then harvest or dissection is used. Flags trigger off these rituals.
+  Flags.add('noWounds', 'You carefully position the corpse and make a cut', 'You bend over .* to make a few quick')
+  
+  ## Checks for status of wound flag in background
+  Thread.new {
+    loop {
+      sleep 3
+      $HUNTING_BUDDY.stop_hunting if Flags['noWounds'] # graceful exit 
+    }
+  }
+
+  
+  ## Run YAML script via combat trainer
+  #DRC.wait_for_script_to_complete('hunting-buddy', ['heal'])
+  
+  ## This section is designed for when running necroheal along with training manager
+  ## It allows a second copy of hunting-buddy to run on top of the current
+  ## hunting-buddy, invoke necroheal, and unpause the original hunting buddy
+
+  pause_script 'hunting-buddy' if running?('hunting-buddy') 
+  force_start_script 'hunting-buddy', [ 'heal' ]
+  
+  matchtimeout 180, "Your inventory" if running?('hunting-buddy')
+  sleep 1 until !running?('combat-trainer')

```

Comment:
If two copies of `combat-trainer` are running (see line 45), will this loop ever end?